MdeModulePkg/Core/Dxe/DxeMain.h

```diff
@@ -1573,6 +1576,48 @@ CoreWaitForEvent (
   OUT UINTN     *UserIndex
   );
 
+// MU_CHANGE begin
+
+/**
+  Stops execution until an event is signaled.
+
+  @param  NumberOfEvents         The number of events in the UserEvents array
+  @param  UserEvents             An array of EFI_EVENT
+  @param  UserIndex              Pointer to the index of the event which
+                                 satisfied the wait condition
+
+  @retval EFI_SUCCESS            The event indicated by Index was signaled.
+  @retval EFI_INVALID_PARAMETER  The event indicated by Index has a notification
+                                 function or Event was not a valid type
+
+**/
+EFI_STATUS
+EFIAPI
+CoreWaitForEventInternal (
+  IN UINTN      NumberOfEvents,
+  IN EFI_EVENT  *UserEvents,
+  OUT UINTN     *UserIndex
+  );
+
+// MU_CHANGE end

```

Comment:
Looks like this MU_CHANGE block should cover the next function prototype.

Reply:
Fixed.